gemfiles/rails_5_2.gemfile

```diff
@@ -4,14 +4,14 @@ source "https://rubygems.org"
 
 gem "rails", "~> 5.2"
 gem "railties", "~> 5.2"
-gem "capybara", "~> 3.0"
+gem "capybara", "~> 3"
 gem "selenium-webdriver", "~> 3.11"
-gem "geckodriver-helper", "~> 0.0.5"
+gem "geckodriver-helper", "~> 0.20.2"
 gem "rspec-rails", "~> 3.7"
 gem "sass-rails", "~> 5.0"
-gem "coffee-rails", "~> 4.2"
+gem "coffee-rails", "~> 4.2.0"
```

Comment:
Is this change and the one below intended here? If so they're a bit varied, so doesn't quite make sense

Reply:
running `appraisal install` did this and I made the assumption it knew what it was doing. will remove though.